docs/usage.rst

```diff
@@ -120,6 +125,11 @@ pseudo-elements::
     >>> selector.css('title::text').get()
     'Example website'
 
+    OR
+
+    >>> selctor.css('title').get(text=True)
+    'Example website'
+
```

Comment:
I think the text parameter needs a proper introduction that indicates how it can be better than `::text` for some scenarios, and maybe feature one of those scenarios (see if the input HTML has any, otherwise consider adding it).

Reply:
So, we need a separate section to illustrate that?